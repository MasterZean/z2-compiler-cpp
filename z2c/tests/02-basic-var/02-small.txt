val a = -0x0s8
val b = 0x0s8
val c = -0x1s8
val d = 0x1s8
val e = -0x64s8
val f = 0x64s8
val g = -0x7Fs8
val h = 0x7fs8
val i = -0x80s8
val j = 0x80s8
val k = -0xC8s8
val l = 0xc8s8