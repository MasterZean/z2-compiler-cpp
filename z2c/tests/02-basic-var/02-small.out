int8 a = 0x00;
int8 b = 0x00;
int8 c = -0x01;
int8 d = 0x01;
int8 e = -0x64;
int8 f = 0x64;
int8 g = -0x7F;
int8 h = 0x7F;
int8 i = -0x80;
DummyClass(10, 9): error:
	'Small' literal constant too big
DummyClass(11, 9): error:
	'Small' literal constant too big
DummyClass(12, 9): error:
	'Small' literal constant too big
